models/backbones/mobilenetv2.py

```diff
@@ -0,0 +1,69 @@
+import torch
+import torchvision
+import torchvision.models as models
+from torchvision.models.mobilenetv2 import InvertedResidual
+
+from .abstract_backbone import AbstractBackbone
+
+
+def layer_props(layer):
+    if isinstance(layer, torchvision.ops.Conv2dNormActivation):
+        # print('Conv2dNormActivation', layer[0].weight.shape, layer[0].stride)
+        input_filters = layer[0].weight.shape[1]
+        stride = layer[0].stride[0]
+        return input_filters, stride
+    elif isinstance(layer, torch.nn.Conv2d):
+        # print('Conv2d', layer.weight.shape, layer.stride)
+        return layer.weight.shape[1], layer.stride[0]
+    elif isinstance(layer, InvertedResidual):
+        # expect
+        input_filters = layer.conv[0][0].weight.shape[1]
+        stride = 1
+        for nestedlayer in layer.conv:
+            _, nested_stride = layer_props(nestedlayer)
+            stride = max(stride, nested_stride)
+        return input_filters, stride
+    else:
+        return 0, 0
+
+
+def get_stride_features_and_filters(model):
```

Comment:
please, add docstring like:
def add(a, b):
    """
    Add two numbers and return the result.

    Args:
        a (int): The first number.
        b (int): The second number.

    Returns:
        int: The sum of the two numbers.
    """
    return a + b

or any other docstring format